blog/src/a-base32-checksum.md

```diff
@@ -0,0 +1,1241 @@
+# A base32 checksum
+
+_(Cue: in the voice of a badly-acted TV commercial.)_
+
+> Do you always mistakenly wire money to or from the wrong account?
+> I used to do that all day, everyday!
+>
+> But then, I found base32check, and my life turned upside-down.
+> base32check is there for me on my daily purchases, watching out for my
+> erroneous transcriptions online and offline! And now, for some reason,
```

Comment:
What's a transcription? Maybe that's a bit too jargon-y

Reply:
Agreed! Especially going in, when you don't know what the subject is. I think `typing errors` is better than `erroneous transcriptions`.

Reply:
Yes!